### PR TITLE
Fix a-la-carte components and eslint for Nuxt v2

### PR DESCRIPTION
**Pull request by XanderLuciano**
I noticed a few issues from @Codefa's PR #56 and spent a little time fixing all the various errors that were appearing in Nuxt v2.

To test this PR, just specify the nuxt2 branch from my repo.

```cmd
$ vue init XanderLuciano/nuxt#nuxt2 my-project
```

Error free a-la-carte :)

image

### CODE REVIEW

**Comment by XanderLuciano** [comment]
This PR will resolve #56, #58, and #59.

As well as resolving issues #54 and #37.

**Comment by XanderLuciano** [comment]
Note: It's recommended to use vuetify-loader for vuetify from 1.3.0 on - https://github.com/vuetifyjs/vuetify/issues/3496#issuecomment-427077294

**Comment by Codefa** [comment]
@XanderLuciano Please merge these fixes to your forked repository let me test your forked vuetify/nuxt repository

**Comment by XanderLuciano** [comment]
Merged to `XanderLuciano/nuxt:master`

You can test with the following command: 

```
$ vue init XanderLuciano/nuxt my-project
```

File: template/assets/style/app.styl [hunk 1]
```diff
@@ -1,8 +1,21 @@
+// Import and define Vuetify color theme
+// https://vuetifyjs.com/en/style/colors
```
Comment (XanderLuciano) [inline template/assets/style/app.styl]: Just realized that the theme does appear to be defined in the plugin, but there is no prompt(s) to enable or disable the lines. 

https://github.com/vuetifyjs/nuxt/blob/master/template/plugins/vuetify.js

```js
Vue.use(Vuetify{{#theme}}, { theme: {
  primary: '#9c27b0',
  accent: '#ce93d8',
  secondary: '#424242',
  info: '#0D47A1',
  warning: '#ffb300',
  error: '#B71C1C',
  success: '#2E7D32'
}}{{/theme}})
```

**Comment by XanderLuciano** [comment]
This was wrong. The context API shown above is separate form the extend build context.

**Comment by Codefa** [comment]
> This was wrong. The context API shown above is separate form the extend build context.

See nuxt starter template is deprecated must use create-nuxt-app

**Comment by XanderLuciano** [comment]
That's fine, I just didn't want to lead any future googlers down the wrong path over such a small misunderstanding and simple 3 line fix.

**Pull request merged**